src/policy/policy.cpp

```diff
@@ -49,7 +49,9 @@ bool IsStandard(const CScript& scriptPubKey, txnouttype& whichType)
             return false;
         if (m < 1 || m > n)
             return false;
-    }
+    } else if (whichType == TX_NULL_DATA &&
+               (!GetBoolArg("-datacarrier", true) || scriptPubKey.size() > nMaxDatacarrierBytes))
```

Comment:
Thanks, I understand your reasoning, and even though it's a bit arbitrary, this may have an impact on applications. For example, a block header doesn't fit in nicely anymore. (I don't really care about those few bytes, but I recall reading about this use case somewhere..)